addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3908,6 +3945,30 @@ const SnippetOptionWidget = Widget.extend({
 
                 return value;
             }
+            case 'setTextWrapping': {
+                const image = this._getImg();
```

Comment:
@visp-odoo  According to me `this._getImg()`
https://github.com/odoo/odoo/blob/e022bdb0eae31db7150d8461c7a97a9a72756472/addons/web_editor/static/src/js/editor/snippets.options.js#L6235 it only retuns this.$target[0]. then why not use it directly. why to call a method for it.